tests/health/gold/sdk/tests__program-path-test.toit.gold

```diff
@@ -0,0 +1,6 @@
+<pkg:pkg-host>/directory.toit:154:6: warning: Deprecated 'platform'
```

Comment:
Why are we getting warnings for a package that we depend on here?

Reply:
The SDK health-check runs with `--show-package-warnings` so we get warnings from inside the SDK itself (which is treated like a package by the compiler).